### PR TITLE
Add salutation in customer and all salutation changes in all views

### PR DESCRIPTION
**Pull request by vijayvargiyakhushboo**
@optimistanoop , @mahesh-maxx  Delete local DB file in your system.I have added 1 new field ('salutation') in customer table.

### CODE REVIEW

File: app/customer/addCustomer.html [hunk 1]
```diff
@@ -3,8 +3,17 @@
 
         <md-content layout-padding layout="row">
           <md-input-container class="md-block" flex-xs>
-            <label>Name</label>
+             <label>salutation</label> 
             <md-icon><i class="material-icons">person</i></md-icon>
+            <md-select name="salutation" ng-model="customer.salutation" ng-required="true">
+             
+             <md-option ng-repeat="salutation in salutation" ng-value="salutation">
```
Comment (optimistanoop) [inline app/customer/addCustomer.html]: same variable two times in "salutation in salutation"?

File: app/transaction/updateTransactionCtrl.js [hunk 2]
```diff
@@ -7,11 +7,19 @@ jhora.controller('updateTransactionCtrl', function($rootScope, $scope, $mdDateLo
...
     $rootScope.editModeData = {};
     $scope.transaction = $scope.editModeData ;
+    $scope.type = '';
+     if($scope.customer.type == 'Mrs'){
+  $scope.type = 'W/o' ;
+ }else if($scope.customer.type == 'Mr.'){
+  $scope.type = 'S/o' ;
+}else{
+  $scope.type = 'D/o' ;
+}
```
Comment (optimistanoop) [inline app/transaction/updateTransactionCtrl.js]: $scope.customer.type?

Comment (vijayvargiyakhushboo) [reply app/transaction/updateTransactionCtrl.js]: I will fix.

File: app/transaction/updateTransactionCtrl.js [hunk 3]
```diff
@@ -7,11 +7,19 @@ jhora.controller('updateTransactionCtrl', function($rootScope, $scope, $mdDateLo
...
 
     $scope.minDate = new Date(new Date().getFullYear() -5, new Date().getMonth(), new Date().getDate());
     $scope.maxDate = new Date();
     $scope.minPromiseDate = $scope.transaction.date ? $scope.transaction.date : new Date();
-    $scope.maxPromiseDate = $scope.transaction.date ? new Date($scope.transaction.date.getFullYear() +1 , $scope.transaction.date.getMonth(), $scope.transaction.date.getDate()) : new Date($scope.transaction.date.getFullYear() , $scope.transaction.date.getMonth() +1 , $scope.transaction.date.getDate());
+   // $scope.maxPromiseDate = $scope.transaction.date ? new Date($scope.transaction.date.getFullYear() +1 , $scope.transaction.date.getMonth(), $scope.transaction.date.getDate()) : new Date($scope.transaction.date.getFullYear() , $scope.transaction.date.getMonth() +1 , $scope.transaction.date.getDate());
```
Comment (optimistanoop) [inline app/transaction/updateTransactionCtrl.js]: why comment here?

Comment (vijayvargiyakhushboo) [reply app/transaction/updateTransactionCtrl.js]: I was fixing date changes.I think due to that it was affected.

**Pull request merged**